assets/scss/modules/tab-block.scss

```diff
@@ -0,0 +1,112 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
```

Comment:
There's a best practice around this module idea that we need to implement in this file. Every css selector in this file should be nested one layer deep under a common parent selector. So for example

```scss
.module {
  .selector-1 {
  
  }
  
  .selector-2 {
  
  }
}
```

A structure like this, replicated across the entire project, ensures that the potential for css conflicts is drastically reduced. For instance another module can also use it's own `.selector-1` class without ever clashing with the one in thie module

Reply:
Neat!  The tab-block-feedback branch will also address this structure.